app/controllers/review_controller.rb

```diff
@@ -91,4 +111,8 @@ def generate_change_note(status, change_note = nil)
   def set_step_by_step_page
     @step_by_step_page = StepByStepPage.find(params[:step_by_step_page_id])
   end
+
+  def require_user_to_be_the_2i_reviewer!
+    render "documents/forbidden", status: :forbidden unless current_user.uid == @step_by_step_page.reviewer_id
```

Comment:
I wonder if this would make more sense in the shared views directory. I think `documents` is a Content Publisher concept.

Reply:
I wondered that too - `shared/` seems to be mostly template partials that can be used to construct pages, rather than containing pages themselves. But agree it would be nice to avoid creating another folder. I'll move it.